tests/testDistortion.py

```diff
@@ -22,1238 +22,99 @@
-
-from collections import namedtuple
-CcdData = namedtuple("CcdData", ['name', 'id', 'corners'])
-CornerData = namedtuple("CornerData", ['focalPlane', 'distEst'])
-
-
-def getVerificationData():
-    """Retrieve verification data
-
-    The verification data consists of the focal plane and distorted positions of
-    each corner of each CCD in HSC.
-
-    The verification data has been pickled for inclusion in this file.  Pickling
-    it is the most convenient means of recording it for posterity, as it is mostly
-    self-contained (apart from the definition of the namedtuple classes, which we
-    prefer to a dict because of their convenience).
-
-    It was constructed with the following code, using hscPipe 3.5.1:
-
-        import pickle
-        import itertools
-        from collections import namedtuple
-        import lsst.afw.geom as afwGeom
-        import lsst.afw.cameraGeom as afwCG
-        from lsst.obs.hsc import HscMapper
-
+    def testDistortion(self):
+        """Test that the distortion data matches the saved data or create new data
+
+        If SAVE_DATA is true then save newly created data and then fail the test
+        in order to prevent anyone from committing the test with SAVE_DATA true!
+
+        Otherwise create new data and compare to the saved data
+        """
+        newData = self.makeDistortionData()
+        dataPath = os.path.join(os.path.dirname(__file__), DataFile)
+
+        if SAVE_DATA:
+            with open(dataPath, "wb") as dataFile:
+                pickle.dump(newData, dataFile, protocol=2)
+            self.fail("Saved new data to %r; please set SAVE_DATA back to False" % dataPath)
+
+        if not os.path.exists(dataPath):
+            self.fail("Cannot find saved data %r; set SAVE_DATA = True and run again to save new data" %
+                      dataPath)
+
+        with open(dataPath, "rb") as dataFile:
+            savedData = pickle.load(dataFile)
+        for detectorName, ccdData in newData.items():
+            savedCcdData = savedData[detectorName]
+            self.assertEqual(ccdData.serial, savedCcdData.serial)
+            for pixPos, cornerData in ccdData.cornerDict.items():
+                savedCornerData = savedCcdData.cornerDict[pixPos]
+                self.assertPairsAlmostEqual(cornerData.focalPlane, savedCornerData.focalPlane)
+                self.assertPairsAlmostEqual(cornerData.pupil, savedCornerData.pupil)
+                self.assertPairsAlmostEqual(cornerData.focalPlaneRoundTrip,
+                                            savedCornerData.focalPlaneRoundTrip)
+                self.assertPairsAlmostEqual(cornerData.pixPosRoundTrip, savedCornerData.pixPosRoundTrip)
+
+    def makeDistortionData(self):
+        """Make distortion data
+
+        If SAVE_DATA is true then the data is persisted before the comparison
+        and the test is then reported as failed in order to prevent anyone
+        from committing the test with SAVE_DATA true!
```

Comment:
This method doesn't actually do any persistence, so suggest removing this note (which is repeated elsewhere multiple times).